### ISSUE TITLE
S3 extract data by date.

### ISSUE THREAD

**Issue by programmeddeath1** [airbytehq/airbyte#15147]
## Tell us about the problem you're trying to solve
I am trying to import data from an S3 datasource, which contains daily generated files from an OLTP database
## Describe the solution you’d like
I wanted to know if it would be possible to make airbyte sync only the new data from the s3 source (i.e. the file with yesterday's date)

## Describe the alternative you’ve considered or used
I am moving the new data to a blank s3 path, and then moving the new file to the complete s3 data lake. Would like to have airbyte just pick the new file from the datalake.

**Comment by artem1205** [comment]
Hi @programmeddeath1 ,
Incremental sync is already implemented in Source S3 Connector. 
This means that you can set a Connection with a daily replication frequency and thus only new records will appear in destination (connector filters out "new" files based on lastModified property.

**Comment by programmeddeath1** [comment]
Yeah but the first time sync tries to replicate the entire existing database which quite huge more than 200Gb

**Comment by artem1205** [comment]
@programmeddeath1 ,
I've added start date field in https://github.com/airbytehq/airbyte/pull/25010, it should help you to filter files

**Comment by artem1205** [comment]
closing as no response
